### PR TITLE
command update in git tutorial

### LINKED ISSUE #1934: Doc update git-data - zarf tools get-admin-password has been replaced
### Environment
Device and OS:
App version:
Kubernetes distro being used:
Other:

### Steps to reproduce
1. https://docs.zarf.dev/examples/git-data/

```
zarf tools get-admin-password
```

```output
NOTE  Using config file /Users/cmwylie19/zarf/zarf-config.toml
Collection of additional tools to make airgap easier

Usage:
  zarf tools [command]

Aliases:
  tools, t

Available Commands:
  archiver         Compresses/Decompresses generic archives, including Zarf packages
  clear-cache      Clears the configured git and image cache directory
  download-init    Downloads the init package for the current Zarf version into the specified directory
  gen-key          Generates a cosign public/private keypair that can be used to sign packages
  gen-pki          Generates a Certificate Authority and PKI chain of trust for the given host
  get-creds        Displays a Table of credentials for deployed components. Pass a component name to get a single credential
  kubectl          Kubectl command. See https://kubernetes.io/docs/reference/kubectl/overview/ for more information.
  monitor          Launches a terminal UI to monitor the connected cluster using K9s.
  registry         Tools for working with container registries using go-containertools
  sbom             Generates a Software Bill of Materials (SBOM) for the given package
  wait-for         Waits for a given Kubernetes resource to be ready

Flags:
  -h, --help   help for tools

Global Flags:
  -a, --architecture string   Architecture for OCI images and Zarf packages
      --insecure              Allow access to insecure registries and disable other recommended security enforcements such as package checksum and signature validation. This flag should only be used if you have a specific reason and accept the reduced security posture.
  -l, --log-level string      Log level when running Zarf. Valid options are: warn, info, debug, trace (default "info")
      --no-log-file           Disable log file creation
      --no-progress           Disable fancy UI progress bars, spinners, logos, etc
      --tmpdir string         Specify the temporary directory to use for intermediate files
      --zarf-cache string     Specify the location of the Zarf cache directory (default "~/.zarf-cache")

Use "zarf tools [command] --help" for more information about a command.
```

### Expected result
```bash
zarf tools get-creds Git
```

output
```bash

 NOTE  Using config file /Users/cmwylie19/zarf/zarf-config.toml
  ✔  Waiting for cluster connection (30s timeout)                                                

 NOTE  Git Server push password (username: zarf-git-user):
ORKmToG2Usr!1WG6uNG0NGFd
```
### Additional Context
Add any other context or screenshots about the technical debt here.

### PR DESCRIPTION
**Pull request by cmwylie19**
## Description
On the git-data [example](https://docs.zarf.dev/examples/git-data/) command `zarf tools get-admin-password` does not exist anymore. Replace this with `zarf tools get-creds Git` which will give the desired password

## Related Issue

Fixes #1934 

Relates to #

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Comment by cmwylie19** [comment]
I think the repo in https://docs.zarf.dev/examples/git-data/ does not exist either, there may be more to do on this PR.

**Comment by cmwylie19** [comment]
also the podinfo example has been removed from the zarf.yaml, im adding it back in

```bash
zarf tools kubectl apply -k http://zarf-git-user:$(zarf tools get-creds Git)@localhost:54824/zarf-git-user/podinfo-1646971829/kustomize

 NOTE  Using config file /Users/cmwylie19/zarf/zarf-config.toml
  ✔  Waiting for cluster connection (30s timeout)                               

 NOTE  Git Server push password (username: zarf-git-user):
service/podinfo created
deployment.apps/podinfo created
horizontalpodautoscaler.autoscaling/podinfo created
```

File: examples/git-data/zarf.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ components:
     required: true
     repos:
       # The following performs a full Git Repo Mirror with `go-git` (internal to Zarf)
-      - https://github.com/defenseunicorns/zarf-public-test.git
```
Comment (Racer159) [inline examples/git-data/zarf.yaml:12]: The tests will need to be updated to properly support this.

**Review by Racer159 — CHANGES_REQUESTED** [review]

File: examples/git-data/zarf.yaml (hunk 1 above)
Comment (cmwylie19) [reply examples/git-data/zarf.yaml:12]: sounds good!

**Review by Racer159 — APPROVED** [review]
lgtm

**Pull request merged**